README.md

```diff
@@ -77,46 +77,49 @@ Checks `.erb` files for syntax errors
 
 Checks `.epp` files for syntax errors.
 
-Note: This will only occur when Puppet version specified is Puppet 4+ or Puppet 3 with the future parser enabled
+Note: This occurs only when the Puppet version specified is Puppet 4 or greater, or Puppet 3 with the future parser enabled.
 
 ## Configuration
 
-Paths can be excluded with:
+[TODO: where does this configuration happen? Is this in the Rakefile, or...?]
+
+* To exclude certain paths:
 
     PuppetSyntax.exclude_paths = ["vendor/**/*"]
 
-When you are using a Puppet version greater then 3.2, you can select the future parse by specifying
+* To use the "future" (Puppet 4) parser in Puppet 3.2 through 3.8, specify:
 
     PuppetSyntax.future_parser = true
 
-If you are using some form of hiera data inside your module, you can configure where the `syntax:hiera:yaml` task looks for data with:
+* To configure specific paths for the Hiera syntax check, set `hieradata_paths`. This is useful if you use Hiera data inside your module.
 
     PuppetSyntax.hieradata_paths = ["**/data/**/*.yaml", "hieradata/**/*.yaml", "hiera*.yaml"]
 
-If you are trying to validate the syntax of code written for application orchestration, you can enable the `app_management` setting:
+* To validate the syntax of code written for application orchestration, enable the `app_management` setting:
 
     PuppetSyntax.app_management = true
 
-`app_management` is supported on Puppet 4.3 or higher, defaulting to off. On Puppet 5, it is always enabled.
+  The `app_management` setting is supported with Puppet 4.3 or greater and is off by default. On Puppet 5, it is always enabled. [TODO: wait, what? Puppet 5 isn't out yet...?]
```

Comment:
enabling app_management by default in Puppet 5 has been widely communicated back when. If you can find it easily, it may be worth verifying, but :man_shrugging: